src/test/kotlin/no/nav/syfo/api/SendPapirSykmeldingManuellOppgaveTest.kt

```diff
@@ -243,10 +297,183 @@ internal class SendPapirSykmeldingManuellOppgaveTest {
                 addHeader(HttpHeaders.Authorization, "Bearer ${generateJWT("2", "clientId")}")
-                setBody(objectMapper.writeValueAsString(smRegisteringManuellt))
+                setBody(objectMapper.writeValueAsString(smRegisteringManuell))
             }) {
                 response.status() shouldEqual HttpStatusCode.NoContent
             }
         }
     }
+
+
+    @Test
+    internal fun `Regsitering av papirsykmelding fra JSON`() {
+        with(TestApplicationEngine()) {
+            start()
+
+            application.setupAuth(
+                VaultSecrets(
+                    serviceuserUsername = "username",
+                    serviceuserPassword = "password",
+                    oidcWellKnownUri = "https://sts.issuer.net/myid",
+                    smregistreringBackendClientId = "clientId",
+                    mqUsername = "username",
+                    mqPassword = "password",
+                    smregistreringBackendClientSecret = "secret",
+                    syfosmpapirregelClientId = "clientid"
+                ), jwkProvider, "https://sts.issuer.net/myid"
+            )
+            application.routing {
+                sendPapirSykmeldingManuellOppgave(
+                    manuellOppgaveService,
+                    kafkaRecievedSykmeldingProducer,
+                    session,
+                    syfoserviceProducer,
+                    oppgaveClient,
+                    kuhrsarClient,
+                    aktoerIdClient,
+                    serviceuserUsername,
+                    dokArkivClient,
+                    regelClient,
+                    syfoTilgangsKontrollClient,
+                    "edbmaskin"
+                )
+            }
+
+            application.install(ContentNegotiation) {
+                jackson {
+                    registerKotlinModule()
+                    registerModule(JavaTimeModule())
+                    configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
+                }
+            }
+            application.install(StatusPages) {
+                exception<Throwable> { cause ->
+                    call.respond(HttpStatusCode.InternalServerError, cause.message ?: "Unknown error")
+                    log.error("Caught exception", cause)
+                    throw cause
+                }
+            }
+
+            coEvery { safDokumentClient.hentDokument(any(), any(), any(), any(), any()) } returns ByteArray(1)
+            coEvery { syfoTilgangsKontrollClient.sjekkVeiledersTilgangTilPersonViaAzure(any(), any()) } returns Tilgang(
+                true,
+                null
+            )
+
+            val oppgaveid = 308076319
+
+            val manuellOppgave = PapirSmRegistering(
+                journalpostId = "134",
+                fnr = "41424",
+                aktorId = "1314",
+                dokumentInfoId = "131313",
+                datoOpprettet = OffsetDateTime.now(),
```

Comment:
Burde bruke OffsetDateTime.now(ZoneId.UTC)

Reply:
Har det så mye å si for testene? Jeg tester ikke spesifikt den funksjonaliteten.

Reply:
Jeg kan gjøre det uansett, for å være konsekvent :)